cluster/swarm/cluster.go

```diff
@@ -241,6 +241,22 @@ func (c *Cluster) RemoveNetwork(network *cluster.Network) error {
 		for _, engine := range c.engines {
 			engine.DeleteNetwork(network)
 		}
+	} else if strings.Contains(err.Error(), "Is the docker daemon running") {
```

Comment:
integration fails.
in the block of `else if`, there is still possibility that `err is nil`,  so `err.Error()` will cause a panic.